scripts/get-model-data.py

```diff
@@ -287,10 +287,11 @@
             'location',
             'publishedAt',
             'requirementsLength',
-            'totalApplications',
+            'contractLength',
             'applicationsFromSMEs',
             'applicationsFromLargeOrganisations',
-            'status_briefs',
+            'totalApplications',
+            'status_data',
```

Comment:
Is this key now coming from the status plus the `data_duplicate_suffix`?

Reply:
Yes, `_briefs` is no longer used.